pkg/utils/utils.go

```diff
@@ -339,3 +384,23 @@ func IsValidHexValue(v string) bool {
 func OpensMenuStyle(str string) string {
 	return ColoredString(fmt.Sprintf("%s...", str), color.FgMagenta)
 }
+
+func MarshalIntoFormat(data interface{}, format string) ([]byte, error) {
+	dataJSON, err := json.MarshalIndent(data, "", "  ")
+	if err != nil {
+		return nil, err
+	}
+	switch format {
+	case "json":
+		return dataJSON, err
+	case "yaml":
+		// Use Unmarshal->Marshal hack to convert vendor-locked objects to YAML with same structure as JSON
```

Comment:
Could you explain a bit more about what's happening here? What do you mean by vendor locked?

Reply:
Sure, `container.Details` declared in `vendor/github.com/docker/docker/api/types/stats.go` already has json tags, but not yaml.

So, in order to keep `./vendor/` files consistent and not changed in any inappropriate way, like
```diff
-- CPUUsage CPUUsage `json:"cpu_usage"`
++ CPUUsage CPUUsage `json:"cpu_usage" yaml:"cpu_usage"`
```
and keep the resulting yaml structure exactly as json is.

Reply:
Ah, that makes sense. Could you capture that in a comment in the code?